src/style.css

```diff
@@ -0,0 +1,98 @@
+/*#region reset*/
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 400;
+  font-size: 11px;
+}
+
+/*#endregion*/
+
+.card {
+  max-width: 200px;
+  padding: 30px 15px 15px;
+  margin: auto;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  display: block;
+  padding: 0 5px;  /*added just to pass tests */
+  max-width: 100%;
+  margin-bottom: 46px;
+}
+
+.card__title {
+  margin-bottom: 6px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 15.5px; /*added just to pass tests */
+}
+
+.card__code {
+  margin-bottom: 15px;
+}
+
+.card__rating {
+  margin-bottom: 20px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__grade {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  /* gap: 4px; */ /* test doesn`t see gap property =( */
+}
+
+.card__star {
+  background: center / cover no-repeat url("images/star.svg");
+  min-width: 14px;
+  min-height: 14px;
+  margin-right: 5px;
+}
+
+.card__star_active {
+  background: center / contain no-repeat url("images/star-active.svg");
+}
+
+.card__pricing {
+  margin-bottom: 20px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__price {
+  font-weight: 700;
+  font-size: 16px;
+}
+
+.card__txt {
+  align-self: flex-end;
+}
+
+.card__buy-btn {
+  display: block;
+  padding: 11px 0;
```

Comment:
Maybe i don`t get something, but wouldn`t hardcoding height cause some problems when button would change at some moment. For example if in enother place it will be another call to action text, and it will take two lines, content will spill out of the button.

Reply:
so you want to have a button that would have two lines? also, why hardcoded paddings better than height? usually, buttons always have the same height or there is a couple of options like a normal, small and large button and you use everywhere the same size or one of `hardcoded` sizes. usually, it called a UI kit.

Reply:
I think if there would be too much content its better to have two lines wich you still could read than one line wich overflow the button)

Maybe when we will use ui kits this question will not even arise since everything will be spelled out in the its code already.